packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -255,6 +255,11 @@ fi
 %endif
 
 %changelog
+
```

Comment:
Version of line 35 also needs to be updated.

Reply:
done